ts_src/index.ts

```diff
@@ -28,7 +28,8 @@ function base (ALPHABET: string): base.BaseConverter {
   const FACTOR = Math.log(BASE) / Math.log(256) // log(BASE) / log(256), rounded up
   const iFACTOR = Math.log(256) / Math.log(BASE) // log(256) / log(BASE), rounded up
 
-  function encode (source: Buffer): string {
+  function encode (source: ArrayLike<number>): string {
```

Comment:
In my opinion, ArrayLike is too permissive at this point.
It allows an object like `{ length: 0 }`.

```suggestion
  function encode (source: Buffer | number[] | Uint8Array): string {
```